espnet2/enh/separator/dptnet_separator.py

```diff
@@ -0,0 +1,189 @@
+from collections import OrderedDict
+from distutils.version import LooseVersion
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+from torch_complex.tensor import ComplexTensor
+
+from espnet2.enh.layers.complex_utils import is_complex
+from espnet2.enh.layers.dptnet import DPTNet
+from espnet2.enh.layers.tcn import choose_norm
+from espnet2.enh.separator.abs_separator import AbsSeparator
+
+is_torch_1_9_plus = LooseVersion(torch.__version__) >= LooseVersion("1.9.0")
+
+
+class DPTNetSeparator(AbsSeparator):
+    def __init__(
+        self,
+        input_dim: int,
+        feature_dim: int = 64,
+        rnn_type: str = "lstm",
+        bidirectional: bool = True,
+        num_spk: int = 2,
+        unit: int = 256,
+        att_heads: int = 4,
+        dropout: float = 0.0,
+        activation: str = "relu",
+        norm_type: str = "gLN",
+        layer: int = 6,
+        segment_size: int = 20,
+        nonlinear: str = "relu",
+    ):
+        """Dual-Path Transformer Network (DPTNet) Separator
+
+        Args:
+            input_dim: input feature dimension
+            feature_dim: intermediate feature dimension for separation
```

Comment:
maybe another name than feature_dim ? bottleneck_dim ?

Reply:
Oh I found that this variable is not used afterwards. So I will remove it.